column.go

```diff
@@ -150,6 +150,9 @@ type Column interface {
 
 	// NotIn creates Condition for "column NOT IN (values[0], values[1] ...)".  Type for values is column's one or other Column.
 	NotIn(values ...interface{}) Condition
+
+	//Is null creates Condiftion for "column IS NULL"
```

Comment:
Minor typos: 
* `Is null` --> `IsNull`
* `Condiftion` --> `Condition`

Reply:
@andrewyap comment has been updated in the latest commit